ui/pages/settings/flask/view-snap/view-snap.js

```diff
@@ -109,138 +108,157 @@ function ViewSnap() {
+    >
+      <Box
+        className="view-snap__header"
+        paddingTop={8}
+        marginLeft={4}
+        marginRight={4}
+      >
+        <SnapAuthorship snapId={snap.id} />
+      </Box>
+      <Box
+        className="view-snap__description"
+        marginTop={4}
+        marginLeft={4}
+        marginRight={4}
+      >
+        <Text variant={TextVariant.bodyMd} color={TextColor.textDefault}>
+          {snap.manifest.description}
+        </Text>
+      </Box>
+      <Box
+        className="view-snap__version_info"
+        marginTop={2}
+        marginLeft={4}
+        marginRight={4}
+      >
+        <Text variant={TextVariant.bodyMd} color={TextColor.textDefault}>
+          {`${t('youInstalled')} `}
+          <span className="view-snap__version_info__version-number">
+            v{snap.version}
+          </span>
+          {` ${t('ofTextNofM')} `}
+          <ButtonLink
+            size={Size.auto}
+            href={url}
+            target="_blank"
+            className="view-snap__version_info__link"
           >
-            {snap.manifest.proposedName}
-          </Typography>
-          <Box className="view-snap__pill-toggle-container">
-            <Box className="view-snap__pill-container" paddingLeft={2}>
-              <SnapAuthorship snapId={snap.id} />
-            </Box>
-            <Box paddingLeft={4} className="view-snap__toggle-container">
-              <Tooltip interactive position="bottom" html={t('snapsToggle')}>
-                <ToggleButton
-                  value={snap.enabled}
-                  onToggle={onToggle}
-                  className="view-snap__toggle-button"
-                />
-              </Tooltip>
-            </Box>
-          </Box>
-        </div>
-        <Box
-          className="view-snap__install-details"
-          display="flex"
-          justifyContent="space-between"
-          alignItems="center"
-          padding={2}
-        >
-          {firstInstall && (
-            <Typography variant={TypographyVariant.H8}>
-              {t('snapAdded', [
-                formatDate(firstInstall.date, 'MMMM d, y'),
-                firstInstall.origin,
-              ])}
-            </Typography>
-          )}
-          <Typography
-            className="view-snap__version"
-            variant={TypographyVariant.H7}
+            {packageName}
+          </ButtonLink>
+          {` ${t('from').toLowerCase()} `}
+          <ButtonLink
+            size={Size.auto}
+            href={firstInstall.origin}
+            target="_blank"
+            className="view-snap__version_info__link"
           >
-            {t('shorthandVersion', [snap.version])}
-          </Typography>
-        </Box>
-        <Box
-          className="view-snap__content-container"
-          width={FRACTIONS.SEVEN_TWELFTHS}
+            {firstInstall.origin}
+          </ButtonLink>
+          {` ${t('on').toLowerCase()} ${formatDate(
+            firstInstall.date,
+            'yyyy-MM-dd',
```

Comment:
Could we use a format that is more widely understood instead?

@eriknson

Reply:
How about going with `16 Nov 2022`, e.g. `DD Mon YYYY`?

Reply:
Updated with proposed format `dd MMM yyyy`.
Examples:
![Screenshot 2023-04-18 at 15 44 32](https://user-images.githubusercontent.com/13301024/232797031-74f41928-e114-44a3-b4fa-81762bab4984.png)
![Screenshot 2023-04-18 at 15 45 22](https://user-images.githubusercontent.com/13301024/232797072-cc23a7d8-8124-4c10-a2ad-fa11a63659fb.png)